xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +212,83 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc\. Wireless Phone (\d{4})$">
+    <description>Cisco Wireless Phone</description>
+    <example hw.model="7925">Cisco Systems Inc. Wireless Phone 7925</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(?i)Cisco Systems, Inc\. IP Phone (CP.{4,10})$">
+    <description>Cisco IP Phone</description>
+    <example hw.model="CP3911">Cisco systems, Inc. IP Phone CP3911</example>
+    <example hw.model="CP-6921">Cisco Systems, Inc. IP Phone CP-6921</example>
+    <example hw.model="CP-7911G">Cisco Systems, Inc. IP Phone CP-7911G</example>
+    <example hw.model="CP-7941G-GE">Cisco Systems, Inc. IP Phone CP-7941G-GE</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Cisco AP (c(?:1130|1140|1240|1250|1260|1600|2600|3500|3600))$" certainty="0.8">
+    <description>Cisco Aironet Wireless Access Point (obsolete)</description>
+    <example hw.model="c1140">Cisco AP c1140</example>
+    <example hw.model="c1260">Cisco AP c1260</example>
+    <param pos="0" name="hw.device" value="WAP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Cisco AP (c(?:1540|1550|1560|1700|1815|1830|1850|2700|2800|3700|3800|701|702))$" certainty="0.8">
+    <description>Cisco Aironet Wireless Access Point</description>
+    <example hw.model="c1830">Cisco AP c1830</example>
+    <param pos="0" name="hw.device" value="WAP"/>
```

Comment:
Here and elsewhere for Aironet:

**_IMO_**,  The value for `hw.model` should not contain the `c` prefix. Cisco doesn't use it in [human consumable marketing](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1830-series-access-points/datasheet-c78-735582.html)  and it enables use to generate CPEs for the devices.

Example hardware CPES:
```
h:cisco:aironet_1815t
h:cisco:aironet_1815w
h:cisco:aironet_1830
h:cisco:aironet_1830e
h:cisco:aironet_1830i
```

Example OS CPE
```
o:cisco:aironet_1800_firmware
o:cisco:aironet_1800i_firmware
o:cisco:aironet_1810_firmware
o:cisco:aironet_1810w_firmware
```

Assuming the example of `Cisco AP c1830` I recommend `hw.model` be `1830` adding the following attributes:

```
    <param pos="0" name="hw.product" value="Aironet {hw.model}"/>
    <param pos="0" name="os.product" value="Aironet {hw.model} Firmware"/>
    <param pos="0" name="os.device" value="WAP"/>
```

Reply:
updated

Reply:
updated